### PR TITLE
Master merge

### PR DESCRIPTION
**Pull request by 0xhank**

### CODE REVIEW

File: contracts/Verifier.sol [hunk 1]
```diff
@@ -404,4 +404,28 @@ library Verifier {
           }
           return verifyProof(a, b, c, inputValues, revealVerifyingKey());
```
Comment (cha0sg0d) [inline contracts/Verifier.sol:405]: Just wondering where there isn't a huge diff here because we changed all the snark keys with our mimc adjustment

**Review by cha0sg0d — COMMENTED** [review]
Looks solid to me, but might find some more errors as we go.

File: contracts/Verifier.sol (hunk 1 above)
Comment (0xhank) [reply contracts/Verifier.sol:405]: i excluded it from the merge

**Review by cha0sg0d — APPROVED** [review]
Test are working, so good to go. We need to seriously update these contracts on the Diamond, but don't want to lose our existing data. so i think an upgrade, not a fresh deploy, is necessary

**Pull request merged**